src/status_im/native_module/core.cljs

```diff
@@ -265,3 +265,12 @@
 
     ;; in unknown scenarios we also consider the device rooted to avoid degrading security
     :else (callback true)))
+
+(defn generate-gfycat
+  [seed]
+  (.generateAlias (status) seed))
+
+(defn identicon
+  [pk]
```

Comment:
isn't it `seed` as well? (if we want to keep it generic, otherwise `public-key` rather than pk to keep naming consistent)

Reply:
we can have seed, although there are some expectations on the format (it needs to be a public key for gfycat actually)

Reply:
if there is expectations could you add  a precondition? and also could you add a docstring for both that explains that it is synchronous calls and what they do?